gost28147_modes.c

```diff
@@ -158,11 +158,14 @@ static int gost28147_cfb_encrypt_segment(struct skcipher_walk *walk,
 		gost28147_cfb_encrypt_one(ctx, iv, dst);
 		crypto_xor(dst, src, bsize);
 		memcpy(iv, dst, bsize);
```

Comment:
This line (`memcpy`) results in tests failure. Please remove it, as done in the original commit.